java/tomcat-native/Portfile

```diff
@@ -16,8 +16,8 @@ long_description    This port provides access to native apr and other \
 homepage            http://tomcat.apache.org/
 master_sites        apache:tomcat/tomcat-connectors/native/${version}/source/
 
-checksums           rmd160  65dc8256c78d3fa2e1f3133c5cf650c108c23b8e \
-                    sha256  1e9409584c19d868efd056f7dbafc767a630a5a95ee3cd516de8b7aaa4cd97b6
+checksums           rmd160  947417c6c0580570e86081931dcad57db2bec4f9 \
+                    sha256  e16858e6ad91c26c17491a26f3ed4a53ab441c44fb3490caf09075ef4dda857e
 
```

Comment:
Please add the new `size` parameter

Reply:
I'm looking for an example of `size` here but not finding it. The only reference to size is part of the `contents` command.

Reply:
Just add it and set it to 1. Then run `sudo port -v checksum` to get the correct value.

Reply:
You can reference other Portfiles too such as py-numpy.

Reply:
Thanks. I have added `size` to the checksums section.